Makefile

```diff
@@ -16,6 +16,16 @@ test:
 	./node_modules/.bin/eslint bouncer/scripts
 	./node_modules/karma/bin/karma start karma.config.js
 
+.PHONY: coverage
+coverage: .coverage
+	@pip install -q tox
+	tox -e coverage
+
+.PHONY: codecov
+codecov: .coverage
```

Comment:
coverage and codecov both depend on the .coverage file but make doesn't know how to make that file, it just crashes if it doesn't exist, you have to run `make test` manually to create it. (If I knew more about makefiles I'd fix this)